relay-dynamic-config/src/defaults.rs

```diff
@@ -73,6 +73,8 @@ fn span_metrics() -> impl IntoIterator<Item = MetricSpec> {
 
     let is_mobile_sdk = RuleCondition::eq("span.sentry_tags.mobile", "true");
 
+    let is_http = RuleCondition::eq("span.op", "http.client");
```

Comment:
Do we need exclusive time metrics for all `http.client` spans? Or is there a way to narrow it down further?